packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -29,17 +29,12 @@ class DeserializationTypeNotFoundException implements Exception {
   String toString() => message;
 }
 
-/// The [SerializableEntity] is the base class for all serializable objects in
+/// The [SerializableModel] is the base interface for all serializable objects in
 /// Serverpod, except primitives.
-abstract mixin class SerializableEntity {
+abstract interface class SerializableModel {
```

Comment:
Discussion: To help with migration, would it make sense to introduce an abstract mixin class named SerializableEntity that we mark as deprecated and note that it will be removed in version 2.1? That way we can give migration instructions that the `SerializableModel` interface should now be used?